src/math/noise.js

```diff
@@ -34,65 +34,141 @@ const scaled_cosine = i => 0.5 * (1.0 - Math.cos(i * Math.PI));
 let perlin; // will be initialized lazily by noise() or noiseSeed()
 
 /**
- * Returns the Perlin noise value at specified coordinates. Perlin noise is
- * a random sequence generator producing a more naturally ordered, harmonic
- * succession of numbers compared to the standard <a href="#/p5/random">random()</a> function.
- * It was invented by Ken Perlin in the 1980s and been used since in
- * graphical applications to produce procedural textures, natural motion,
- * shapes, terrains etc.<br /><br /> The main difference to the
- * <a href="#/p5/random">random()</a> function is that Perlin noise is defined in an infinite
- * n-dimensional space where each pair of coordinates corresponds to a
- * fixed semi-random value (fixed only for the lifespan of the program; see
- * the <a href="#/p5/noiseSeed">noiseSeed()</a> function). p5.js can compute 1D, 2D and 3D noise,
- * depending on the number of coordinates given. The resulting value will
- * always be between 0.0 and 1.0. The noise value can be animated by moving
- * through the noise space as demonstrated in the example above. The 2nd
- * and 3rd dimensions can also be interpreted as time.<br /><br />The actual
- * noise is structured similar to an audio signal, in respect to the
- * function's use of frequencies. Similar to the concept of harmonics in
- * physics, Perlin noise is computed over several octaves which are added
- * together for the final result. <br /><br />Another way to adjust the
- * character of the resulting sequence is the scale of the input
- * coordinates. As the function works within an infinite space the value of
- * the coordinates doesn't matter as such, only the distance between
- * successive coordinates does (eg. when using <a href="#/p5/noise">noise()</a> within a
- * loop). As a general rule the smaller the difference between coordinates,
- * the smoother the resulting noise sequence will be. Steps of 0.005-0.03
- * work best for most applications, but this will differ depending on use.
+ * Returns random numbers that can be tuned to feel more organic. The values
```

Comment:
Two points of feedback for this one:
- Should we mention briefly how it differs from the random numbers in random()? Maybe by describing it as smooth?
- I think one piece of info from before that's useful to know is the output range (e.g. so you can use map() on it)

Reply:
Agreed on both points.